main.go

```diff
@@ -104,6 +104,8 @@ func init() {
 	viper.BindPFlag("hooks", rootCmd.PersistentFlags().Lookup("hooks"))
 	rootCmd.PersistentFlags().String("proc_path", "/proc", "Proc mount path")
 	viper.BindPFlag("proc_path", rootCmd.PersistentFlags().Lookup("proc_path"))
+	rootCmd.PersistentFlags().Int("registering_retry", 100, "Executor registering delay in milliseconds")
```

Comment:
Please use "Duration" type instead of int. It allows user to set the wanted unit, eg: 3s or 100ms.